pep-0664.rst

```diff
@@ -81,7 +81,19 @@ Features for 3.11
 
 Some of the notable features of Python 3.11 include:
 
- ** Watch this space :) **
+* :pep:`654`, Exception Groups and ``except*``.
+* :pep:`657`, Enhanced error locations in tracebacks.
+* :pep:`680`, Support for parsing TOML in the standard library
+* A ~1.25x speedup from 3.10
```

Comment:
This is a bit simplistic. Please, preserve the original text (mentioning that "Python 3.11 is up to 10-60% faster than Python 3.10. On average, we measured a 1.22x speedup on the standard benchmark suite") as much as possible and link against https://docs.python.org/3.11/whatsnew/3.11.html#faster-cpython.

Reply:
@pablogsal isn't it meant to be short snippets? I guess we can add a link

Reply:
> @pablogsal isn't it meant to be short snippets

There is no specific requirements, is mainly what the RM decides ;)

Reply:
valid. hows this?
> Python 3.11 is up to 10-60% faster than Python 3.10. On average, we measured a 1.25x speedup on the standard benchmark suite. See [Faster CPython](https://docs.python.org/3.11/whatsnew/3.11.html#faster-cpython) for details.

(from [here](https://docs.python.org/3.11/whatsnew/3.11.html#summary-release-highlights))

Reply:
Looks good 👍

Reply:
```suggestion
* Python 3.11 is up to 10-60% faster than Python 3.10. On average, we measured
  a 1.25x speedup on the standard benchmark suite. See `Faster CPython 
  <https://docs.python.org/3.11/whatsnew/3.11.html#faster-cpython>`__
  for details.
```